test/src/debugInfo.spec.ts

```diff
@@ -24,11 +24,8 @@ describe('DebugInfo', function () {
         });
       }
 
-      if (browser.debugInfo.pendingProtocolErrors.length) {
-        throw new Error(
-          'browser.debugInfo.pendingProtocolErrors is not empty before the test'
-        );
-      }
+      // Insure that the previous test are flushed
+      expect(browser.debugInfo.pendingProtocolErrors).toHaveLength(0);
```

Comment:
Let's put the context of the browser.debugInfo.pendingProtocolErrors into the error message?